packages/frontend/src/view/pages/user/components/UserQuickActionsTable.tsx

```diff
@@ -50,7 +55,17 @@ export function UserQuickActionsTable(props: UserQuickActionsTableProps) {
                 </InlineEllipsis>
               </strong>
             </p>
-            <Button className="ml-auto w-32 !px-0">Withdraw now</Button>
+            {props.user && asset.asset.details && (
+              <UserWithdrawNowButton
+                className="ml-auto w-32 !px-0"
+                assetDetails={asset.asset.details}
+                account={props.user.address}
+                starkKey={props.starkKey}
+                exchangeAddress={EthereumAddress.fake()}
```

Comment:
Is this intentional?

Reply:
yeah, I did not know what to put there, but it is going to be changed